modules/internal/ChapelRange.chpl

```diff
@@ -166,6 +166,17 @@ module ChapelRange {
   pragma "no doc"
   config param useOptimizedRangeIterators = true;
 
+  /* Chapel is in the process of changing ``range(idxType).size`` away
+     from returning the range's size as an ``idxType`` value in favor
+     of returning an ``int`` value.  Setting ``sizeReturnsInt`` to
+     ``true`` permits a user to opt into this new behavior now rather
+     than having it change out from under them in a future release.
+     The old behavior can be retained by using the new
+     :proc:`range.sizeAs` method.  */
+
+  config param sizeReturnsInt = false;
```

Comment:
Seems slightly odd for this to be in ChapelRange when it applies to ranges/domains/arrays. I wonder if chapelbase or something else makes more sense, but we probably don't chpldoc chapelbase.

Reply:
I think you're right that we don't (document ChapelBase), and think that we should eventually.

I don't think it's out of place here in the sense that arrays depend on domains and domains depend on ranges, so it feels at an appropriate point in the code hierarchy.  It's a bit odder in terms of the documentation hierarchy, but since it's a short-term feature by design, and one that hopefully most users won't ever have to use (because they use `int` idxTypes in their code almost all of the time... fingers crossed), I don't feel too sheepish about it being here.  And I think I got the docs in the other modules to correctly cross-link to the documentation in ChapelRange such that people will find their way here when need be (?).

Reply:
Sounds good.